### PR TITLE
Added tags to Tag_definitions.csv

### PR DESCRIPTION
**Pull request by bhg**
Added tags from Issue #465 into Tag_definitions.csv.

Tested by running the admin console and checking that all new tags were present and correct.

### CODE REVIEW

File: common/models/Tag_definitions.csv [hunk 1]
```diff
@@ -272,11 +279,12 @@ hack-for-your-mother-2019,Hack for Your Mother 2019,,Organization,,
 hack-for-your-mother-2020,Hack for Your Mother 2020,,Organization,,
 summer-solve-a-thon-2019,Summer Solve-a-thon 2019,,Organization,,
 nonprofit,501(c)(3) Nonprofit,,Organization Type,,
-other-nonprofit,Other 501(c) Nonprofit,,Organization Type,,
+other-nonprofit,Other Nonprofit,,Organization Type,,
+government,Government,,Organization Type,,
```
Comment (marlonkeating) [inline common/models/Tag_definitions.csv:283]:
```suggestion
government-org,Government,,Organization Type,,

```

File: common/models/Tag_definitions.csv [hunk 2]
```diff
@@ -215,6 +221,7 @@ ui-designer,UI Designer,,Role,Design,
 graphic-designer,Graphic Designer,,Role,Design,
 creative-director,Creative Director,,Role,Design,
 design-lead,Lead Designer,,Role,Design,
+ui-copywriter,UI Copywriter,,Role,Design,
```
Comment (marlonkeating) [inline common/models/Tag_definitions.csv:224]:
```suggestion
ux-copywriter,UX Copywriter,,Role,Design,

```

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Just a couple adjustments to make, partially due to typos on the original issue description.

**Comment by bhg** [comment]
Changes made as suggested.

**Review by marlonkeating — CHANGES_REQUESTED** [review]
Text changes look fine, but the extra whitespaces that got added cause the tag import to fail.

File: common/models/Tag_definitions.csv [hunk 3]
```diff
@@ -215,6 +221,8 @@ ui-designer,UI Designer,,Role,Design,
 graphic-designer,Graphic Designer,,Role,Design,
 creative-director,Creative Director,,Role,Design,
 design-lead,Lead Designer,,Role,Design,
+ux-copywriter,UX Copywriter,,Role,Design,
+
```
[the remark this answers is not in the document]
Comment (bhg) [reply common/models/Tag_definitions.csv]: Fixed now, and checked admin console.

**Review by marlonkeating — APPROVED** [review]

**Pull request merged**